.github/workflows/ci.yml

```diff
@@ -352,6 +352,49 @@ jobs:
               body: 'Failed to pass the UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
             })
 
+    build_static_html_composites_sample:
+    name: Build And Test Component+Binding Examples
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Install rush
+        run: npm install -g @microsoft/rush@5.47.0
+      - name: Install dependencies
+        run: rush install
+      - name: Build
+        run: rush build -t component-examples
+      - name: Visual Regression Tests
+        id: visualregressiontests
+        run: |
+          cd samples/tests
+          rushx test:e2e:examples
+        env:
+          CONNECTION_STRING: ${{ secrets.CONNECTION_STRING }}
+      - name: Upload snapshot diff
+        if: ${{ always() && steps.visualregressiontests.outcome == 'failure' }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: snapshots
+          path: samples/tests/test-results/ # or path/to/artifact
+      - name: Comment on PR
+        if: ${{ github.event_name == 'pull_request' && always() && steps.visualregressiontests.outcome == 'failure' && !contains( github.event.pull_request.labels.*.name, 'ui change') }}
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Failed to pass the UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
+            })
```

Comment:
I wonder if we could extract some common logic here? And make something like a function? @JamesBurnside I guess you are the only expert of us might have some answers for this
But it is not this PR related, will need to open a work item for this

Reply:
GH released workflow sharing ability recently, but I've heard mixed reviews of that so far.

Could we make the `body` here (and in other UI tests) more concrete now? There are multiple kinds of failures that all add the same message.